### DISCUSSION TITLE
Failure to boot ‘Install macOS Monterey’

### DISCUSSION THREAD

**Discussion by ghost** [SkyrilHD/Dell-E7250-Hackintosh#56] [General]
Hello, hope you're well.
After successfully creating install media (via Terminal) there's an issue during boot up. The load bar stops at ~%20 and the computer restarts after that. Resetting NVRAM does not fix this. Also my install drive does not appear in Startup Disk.

**Comment by SkyrilHD** [comment]
Make sure you formatted the drive correctly (APFS, GPT). Also, enabling verbose mode might help as well. You can do so by adding -v inside boot-arg (config.plist)

**Reply by ghost** [reply]
I checked those things. They're correct. Do you think formatting my internal SSD and installing Monterey to it is the next step?

macOS Big Sur worked since day one with your (at that time) latest EFI. I don't know why my Dell E7250 is failing to boot Monterey. I have OC experience but not enough to fix this seemingly.
